includes/scl-preview.md

```diff
@@ -5,5 +5,5 @@ ms.date: 05/22/2022
 ms.topic: include
```

Comment:
This should move to live with the commandline docs `docs/standard/commandline/includes/scl-preview.md`

Reply:
@adegeo is suggesting moving this file _to_ that location so that all the commandline stuff is together.

Reply:
Thanks. If that's not a problem I am not going to do that right now, as we want to publish the docs today.